src/client/eppo-precomputed-client.spec.ts

```diff
@@ -702,7 +702,7 @@ describe('EppoPrecomputedClient E2E test', () => {
       client.setSubjectSaltAndPrecomputedFlagStore(
         'test-subject',
         { attr1: 'value1' },
-        encodeBase64(salt),
+        salt,
```

Comment:
There's a bunch of other calls to `encodeBase64` down the file

Reply:
✔️